compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -37,13 +38,59 @@
     from pika.spec import BasicProperties
 
 
+# List of Linux capabilities that would suggest the running process is
+# privileged, even if os.getuid() != 0.
+_MULTI_USER_CAPS = {
```

Comment:
I assume this applies for everything except Windows, which we don't support anyway?

Reply:
No, it applies only to Linux.  For MVP, clusters&nbsp;&mdash;&nbsp;specifically _Linux_ clusters&nbsp;&mdash;&nbsp;are the target.

There's already some issue with the code on macOS which uses a different security model; the Python modules don't appear to have the necessary functions (i.e., the `os.set*id()` functions, as tested with Yadu a few months ago).  So, will need to work on that, but that's going to be in a future development effort.